CMakeLists.txt

```
@@ -277,10 +277,16 @@ if(SENTRY_BACKEND_CRASHPAD)
 	target_compile_features(sentry PUBLIC cxx_std_14)
 	target_link_libraries(sentry PRIVATE crashpad_client crashpad_util)
 	if(NOT BUILD_SHARED_LIBS)
-		sentry_install(TARGETS crashpad_client crashpad_util crashpad_compat getopt mini_chromium zlib EXPORT sentry
+		sentry_install(TARGETS crashpad_client crashpad_util crashpad_compat mini_chromium EXPORT sentry
 			LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
 			ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
 		)
+		if(WIN32)
```

https://github.com/getsentry/crashpad/blob/4ec46c337905a10f1690ef57594daa11d6e08644/CMakeLists.txt#L91-L94

I think it should be:
```suggestion
		if(MSVC)
```

---

Good catch. I was using the version of crashpad's `CMakeLists.txt` from the `0.2.2` release and that used to say `IF(WIN32)` there. I will update to say `IF(MSVC)` here for consistency with `master`.